runtime/src/main/scala/tailcall/runtime/service/GraphQLGenerator.scala

```diff
@@ -36,7 +40,7 @@ object GraphQLGenerator {
         override protected val wrappers: List[Wrapper[Any]]                  = Nil
         override protected val additionalDirectives: List[__Directive]       = Nil
         override protected val features: Set[Feature]                        = Set.empty
-      }
+      }.withAdditionalTypes(additionalTypes)
```

Comment:
@tusharmath - found a workaround for the missing types problem by adding all the types in `withAdditionalTypes` 
This caused some tests in `Config2SDLSpec` to break (which seemed to be assuming the un-referenced types being missing) which I had to fix here.
https://github.com/tailcallhq/tailcall/pull/176/files#diff-448e95d2e72ed1953ebd5b75ed14f122b5724dab43aca904a6a12f4b5e6a06c0

Reply:
We can just add the types that are being extended using the extends annotation. I think that will make sure only those aren't remmove.

Reply:
Thx! Updated the code to add only the types from the interfaces.